internal/distro/rhel9/images.go

```diff
@@ -111,6 +111,9 @@ func osCustomizations(
 			},
 		)
 	}
+	if imageConfig.Keyboard != nil && imageConfig.Keyboard.X11Keymap != nil {
+		osc.X11Layouts = imageConfig.Keyboard.X11Keymap.Layouts
+	}
```

Comment:
This should be moved to the block above:
```go
if keyboard != nil {
	osc.Keyboard = keyboard
} else if imageConfig.Keyboard != nil {
	osc.Keyboard = &imageConfig.Keyboard.Keymap
}
```
The whole config (keymap and x11layouts) are currently applied as a whole - https://github.com/osbuild/osbuild-composer/blob/3443484502b448fd00c5596ab546c190be781518/internal/distro/rhel9/pipelines.go#L420-L424

Lastly, the `X11Keymap.Layouts` tend to be related to the value set in `Keyboard`.

Reply:
The way it's done now (see block below) allows for a customization to change the base `Keymap` but still maintain the `X11Keymap.Layouts` if the image config defines them.
Indeed, they are related, but since we don't support customizing the ``X11Keymap` (yet?), I wasn't sure whether an external customization for the base `Keymap` should override it or not.

```go
	if keyboard != nil {
		osc.Keyboard = keyboard
	} else if imageConfig.Keyboard != nil {
		osc.Keyboard = &imageConfig.Keyboard.Keymap
	}
	if imageConfig.Keyboard != nil && imageConfig.Keyboard.X11Keymap != nil {
		osc.X11KeymapLayouts = imageConfig.Keyboard.X11Keymap.Layouts
	}
```

Reply:
I guess keeping the image config can lead to weird issues where you have an X11KeymapLayout that's a whole different thing from the base Keymap, so it makes sense to override completely.  I'll change it.

Reply:
Yeah, my point was that the image config values (all of them) related to keyboard settings should be thrown away if the user specified anything in the Blueprint. At least that's how it works now and IMO it makes sense. The reason is that the X11Keymap layout tends to be related to the Keymap value, so overriding just one of them with user-provided value and applying the other as it was set in image config does not IMO make sense.